seed/challenges/html5-and-css.json

```diff
@@ -3775,8 +3775,8 @@
         "Replace the word <code>black</code> in our <code>body</code> element's background-color with its <code>hex code</code> representation, <code>#000000</code> "
       ],
       "tests": [
-        "assert($(\"body\").css(\"background-color\") === \"rgb(0, 0, 0)\", 'message: Give your <code>body</code> element the background-color of black.');",
-        "assert(code.match(/#000000/g) && code.match(/#000000/g).length > 0, 'message: Use the <code>hex code</code> for the color black instead of the word <code>black</code>. For example <code>body { color&#58; #000000; }</code>');"
+        "assert($(\"body\").css(\"background-color\") === \"rgb(0, 0, 0)\" && code.match(/body\\s?{([\\s\\S]*?)}/i).toString().indexOf(\"background\") != -1, 'message: Give your <code>body</code> element the background-color of black.');",
+        "assert(code.match(/body\\s?{([\\s\\S]*?)}/i)[0].match(/background.*/i)[0].match(/(^#|#)([a-zA-Z0-9]+)/gi).toString().toUpperCase() === \"#000000\" || code.match(/body\\s?{([\\s\\S]*?)}/i)[0].match(/background.*/i)[0].match(/(^#|#)([a-zA-Z0-9]+)/gi).toString().toUpperCase() === \"#000\", 'message: Use the <code>hex code</code> for the color black instead of the word <code>black</code>. For example <code>body { color&#58; #000000; }</code>');"
```

Comment:
> `match(/body\s*{[\\s\\S]*?background.*\s*:\s*#000(000)?;[\\s\\S]*?}/gi)`

Ah thanks, I knew it was possible. I couldn't translate the words in my mind to a term for the searchengine. I ended up doing seperate checks :(.